README.md

```
@@ -24,16 +24,25 @@ A bidirectional bridge that connects Mattermost and Matrix, enabling real-time m
 
-### 2. Configure Settings
+### 2. Register a Matrix homeserver
 
-- Go to System Console → Plugins → Matrix Bridge
-- Set your Matrix homeserver URL (e.g., `https://matrix.example.com`)
-- Generate Application Service and Homeserver tokens
-- Enable message synchronization
+Homeservers are managed entirely through `/matrix server` slash commands (System Admin
+only) - there is no System Console configuration for them:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)